ep/build-velox/src/get_velox.sh

```diff
@@ -106,15 +106,15 @@ function process_setup_ubuntu {
   fi
   sed -i "s/apt install -y/sudo apt install -y/" ${VELOX_HOME}/scripts/setup-adapters.sh
   if [ $ENABLE_S3 == "ON" ]; then
-    sed -i '/^  run_and_time install_folly/a \ \ '${VELOX_HOME}/scripts'/setup-adapters.sh aws' scripts/setup-ubuntu.sh
+    sed -i '/^  run_and_time install_folly/a \ \ SUDO=$SUDO \n '${VELOX_HOME}/scripts'/setup-adapters.sh aws' scripts/setup-ubuntu.sh
```

Comment:
Use `SUDO=$SUDO <command>` instead of `export SUDO <command>`. So env `SUDO` is only passed to current command.